doc/extension_store/README.md

```diff
@@ -0,0 +1,74 @@
+# Proposal for Extension Store Manifest
+
+Example for manifest: [manifest.jsonc](./manifest.jsonc)
+
+## Variable Replacements
+
+* Variables placeholders use the `"My variable 'variableName' has value '${variableName}'"` syntax
+* Naming conventions:
+  * User defined variables are in `lowerCamelCase`
+  * Global variables defined by the environment are in `UPPER_CASE`, e.g. the BucketFS service and Bucket where to install files.
+* User defined variables are defined in the `parameters` section. The Extension Store asks the user to input values for all parameters.
+* Variable placeholders in strings (e.g. BucketFS paths or SQL scripts) are expanded before using them.
+
+## Parameters to Store
+
+The Extension Store must store the following information for an installed VS:
+
+* For each adapter:
+    * ID and version
+        * Required to detect if an older version is already installed
+        * Required to detect if the user tries to install another VS using the same adapter
+    * Paths to installed JARs on BucketFS
+        * File names might change in newer versions.
+        * Needed to delete files from BucketFS during update/delete
```

Comment:
No. We store the specific paths and delete only those.
But if you prefer we can also keep all files for an adapter in a single directory.

Reply:
If you allow arbitrary path, you might get into trouble if extensions use the same path. Then you make sure, that not only the name is equal, but also the content otherwise you can introduce errors.

Reply:
Changed this to have a common prefix for an extension version